src/prob/ne.jl

```diff
@@ -9,7 +9,7 @@ end
 
 # construct the gas flow feasbility problem with demand being the cost model
 function build_ne(pm::_PM.AbstractPowerModel, gm::_GM.AbstractGasModel; kwargs...)
-    kwargs = Dict(kwargs)
+    kwargs = Dict(kwargs[:kwargs])
```

Comment:
Did you try, `kwargs = Dict(kwargs...)`?

If we have time let's discuss if there is a simpler way to implement this.

Reply:
Per our discussion, objective weights will live in `PowerModels` input files as global parameters.